pkg/lib/hooks.js

```diff
@@ -325,3 +325,21 @@ export function useEvent(obj, event, handler) {
 export function useInit(func, deps, comps) {
     return useObject(func, null, deps || [], comps);
 }
+
+/* XXX - document this
+ */
+
+export function useMockConfig() {
+    useEvent(window, "test_mock_changed");
+    return window.test_mock;
+}
+
+export function useMock(topic) {
+    const config = useMockConfig();
+
+    return (val, mock_val) => {
+        if (config === true || (topic && config === topic))
+            return mock_val || "mock";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17654-20220822-064711-e2f09928-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)